src/sign-in-webapp/README.md

```diff
@@ -69,12 +69,20 @@ Later the web app can make calls to the Microsoft 365 Graph API on behalf of sig
 
 ## Scaffold the web app by using the ASP.NET Core Razor project template
```

Comment:
New template coming in a few minutes. In the README, they'll learn how to prepare for and set up the sample app (installing packages if they need to) but we won't have them scaffold a _new_ application in the README.

Reply:
thanks for creating that template @mmacy, we're addressing all readme-related changes from another [pr](#9).